### ISSUE TITLE
Avocode crashed after unseccessful layer selection

### ISSUE THREAD

**Issue by andrew-mykhalchuk** [avocode/avocode#2209]
This is additional issue that happened following https://github.com/avocode/avocode/issues/2208
---
Here's what happened this time:
1. I restarted the app.
2. Attempted to select any layer again - received errors described in linked issue above.
3. After multiple unsuccessful attempts app crashed and suggested opening a new issue and this one seems to be different so I decided not to combine them into one.
4. Other designs (at least originally made in Sketch) are working just fine.

Yes, I'm on the latest version

---
* **Version**: 3.6.0
* **Web/Desktop**: Desktop
* **Platform**: darwin
---
```
{ preventDefault: [Function: preventDefault],
  sender:
   WebContents {
     webContents: [Circular],
     history:
      [ 'file:///Applications/Avocode.app/Contents/Resources/app.asar/desktop-index.html?appDataDirname=%2FUsers%2Fandrew_mykhalchuk%2F.avocode' ],
     currentIndex: 0,
     pendingIndex: -1,
     inPageIndex: -1,
     _events:
      { 'navigation-entry-commited': [Function],
        'ipc-message': [Function],
        'ipc-message-sync': [Function],
        'pepper-context-menu': [Function],
        '-did-get-response-details': [Function],
        '-did-get-redirect-request': [Function],
        'devtools-reload-page': [Function],
        'will-navigate': [Array],
        'did-navigate': [Function],
        destroyed: [Function],
        'devtools-opened': [Function],
        '-new-window': [Function],
        '-web-contents-created': [Function],
        '-add-new-contents': [Function],
        move: [Function],
        activate: [Function],
        'page-title-updated'
```
---
```
["app-finished-launching","app-crash-reporter-starting","app-crash-reporter-started","appready","appopenlisteners-replacing","appopenlisteners-replaced","appsplash-creating","appsplash-created","createwindow-starting","createwindow-browser-window-preparing","createwindow-browser-window-prepared","createwindow-crash-listener","createwindow-state-manage","createwindow-load-url","createwindow-dom-ready-listener","createwindow-finishing","ipc--calling-methog--setTitle","mainwindow-dom-ready","mainwindow-destroying-splash","mainwindow-showing","mainwindow-shown","mainwindow-opening-init-urls","mainwindow-opening-init-files"]
```

**Comment by crashmajstr** [comment]
Hi, please, can you tell me if the error poped out only when you inspected PSDs? @mehocode

**Comment by crashmajstr** [comment]
Hi, just wanna inform you that we’ve just released the new version of Avocode 3.6.1 which could fix this issue. 
Please, update your desktop app, if you use the web app the new version is already there.
Let me know if the problem still persists or not. Thank you

**Comment by andrew-mykhalchuk** [comment]
Hi @crashmajstr - I can confirm this is now fixed. I'm on v3.6.2 already. Appreciate blazing fast updates!

Thank you
